MekHQ/src/mekhq/campaign/parts/MissingAvionics.java

```diff
@@ -75,6 +76,24 @@ public int getDifficulty() {
 
     @Override
     public String checkFixable() {
+        if ((unit != null) && (unit.getEntity() instanceof LandAirMech)) {
+            // Avionics are installed in the Head and both Torsos,
+            // make sure they're not missing.
+            for (Part part : unit.getParts()) {
```

Comment:
I don't suppose there's a better way to do this rather than looping through every part in the unit?

Reply:
Not without a bit of work on `Unit` and I'm waiting til I have good tests for that.